test/autID.test.js

```diff
@@ -31,7 +31,7 @@ describe("AutID", function () {
       await daoTypes.deployed();
 
       const SWLegacyMembershipChecker = await ethers.getContractFactory(
-        "SWLegacyMembershipChecker"
+        "TributeMembershipChecker"
       );
```

Comment:
Can you please leave the SWLEgacyMembershipChecker here? 

Having tests on the TributeMembershipChecker is enough.

Reply:
Done